rules/utils.go

```diff
@@ -427,42 +572,22 @@ var validMachineSizes = []string{
 	"Standard_FX36mds",
 	"Standard_FX48mds",
 	"Standard_FX4mds",
-	"Standard_G1",
-	"Standard_G2",
-	"Standard_G3",
-	"Standard_G4",
-	"Standard_G5",
-	"Standard_GS1",
-	"Standard_GS2",
-	"Standard_GS3",
-	"Standard_GS4",
-	"Standard_GS4-4",
-	"Standard_GS4-4",
-	"Standard_GS4-8",
-	"Standard_GS4-8",
-	"Standard_GS5",
-	"Standard_GS5-16",
-	"Standard_GS5-16",
-	"Standard_GS5-8",
-	"Standard_GS5-8",
-	"Standard_H16",
-	"Standard_H16m",
-	"Standard_H16mr",
-	"Standard_H16r",
-	"Standard_H8",
-	"Standard_H8m",
```

Comment:
Why are these removed?

Reply:
1. they are not returned by the API.
2. They are retired, see https://learn.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs#h-series

> On August 31, 2022, we're retiring the following H-series Azure VM sizes:
> 
> H8
> H8m
> H16
> H16r
> H16m
> H16mr
> H8 Promo
> H8m Promo
> H16 Promo
> H16r Promo
> H16m Promo
> H16mr Promo

Would you suggest to keep them?  I can update the PR.

Reply:
@wata727 ping

Reply:
Thank you for your explanation.
I was concerned if anyone was still using these sizes, but according to the article, it seems that no such user exists anymore.